### PR TITLE
Fix text input copy paste duplication

### PR DESCRIPTION
**Pull request by ethantran**
**Description:**

- [Asana Task](https://app.asana.com/0/1138529592199337/1165205282200433)

- [x] Have you read the [contributing guidelines](https://eds.ethoslabs.io/#/Guidelines?id=section-contribute)?
- [x] Followed the checklist at the bottom of the [contributing guidelines](https://eds.ethoslabs.io/#/Guidelines?id=section-contribute) guide?
- [x] Bumped the yarn version?

### CODE REVIEW

File: src/components/TextAreaInput/TextAreaInput.js [hunk 1]
```diff
@@ -88,12 +88,6 @@ function PrivateTextAreaInput({
     doValidation(ev.target.value, true)
   }
 
-  const onPaste = (ev) => {
-    const val = ev.clipboardData.getData('text/plain')
-    const restrictedVal = restrictIllegal ? restrict(val) : val
-    setValue(restrictedVal)
-  }
-
```
Comment (ethantran) [inline src/components/TextAreaInput/TextAreaInput.js]: Every onPaste event was was followed by an onChange event, causing setValue with restrictedVal twice.

Comment (ethantran) [reply src/components/TextAreaInput/TextAreaInput.js]: Pasting without the onPaste handler still calls onChange so the sanitization still occurs.

Comment (stephaniel96) [reply src/components/TextAreaInput/TextAreaInput.js]: :nod:

**Comment by stephaniel96** [comment]
I think you might want to update the `onPaste` prop for `BaseTextAreaInput` as well

**Comment by ethantran** [comment]
yep no problems with base input

**Comment by roblevintennis** [comment]
Thanks Ethan!

**Pull request merged**